scripts/ci/tags.yaml

```diff
@@ -1240,6 +1240,7 @@ ci_tests_benchmarks_current_consumption:
   files:
     - modules/hal/nordic/nrfx/
     - nrf/tests/benchmarks/current_consumption/
+    - zephyr/drivers/gpio
```

Comment:
I will conflict with https://github.com/nrfconnect/sdk-nrf/pull/17559, maybe you can skip that update?